desktop/autoUpdateListeners.js

```diff
@@ -6,6 +6,8 @@ import { ipcConsts } from '../app/vars';
 const subscribeToAutoUpdateListeners = ({ mainWindow }) => {
   autoUpdater.on('error', (error) => mainWindow.webContents.send(ipcConsts.WALLET_UPDATE_ERROR, { error }));
 
+  autoUpdater.on('download-progress', (progressObj) => mainWindow.webContents.send(ipcConsts.DOWNLOAD_UPDATE_PROGRESS, { downloadPercent: progressObj.percent }));
```

Comment:
percentage is not used as progress indicator so there is no sense in sending it as number. Also very frequent ipc communication will cause performance degradation as it happened in earlier version of update download progress. Find a way to send this once as boolean flag